prawcore/sessions.py

```diff
@@ -195,6 +197,9 @@ def _make_request(
             log.debug(
                 f"Response: {response.status_code}"
                 f" ({response.headers.get('content-length')} bytes)"
+                f" ({response.headers.get('x-ratelimit-reset')}:"
+                f"{response.headers.get('x-ratelimit-remaining')}:"
+                f"{response.headers.get('x-ratelimit-used')} ratelimit) at {time.time()}"
```

Comment:
Can we add names for these to the log entry?

Reply:
What would you suggest? I was trying to not increase the size of the log message by a bunch.

Reply:
rst, rem, used?

Reply:
Added